rebase.sh

```
@@ -13,6 +13,9 @@
 set -e
 set -u
 
+PREVIOUS_UPSTREAM_VERSION="release/1.104"
```

Shellcheck (0.11.0)

[warning] PREVIOUS_UPSTREAM_VERSION appears unused. Verify use (or export if used externally).

(SC2034)